kivy/uix/behaviors/compoundselection.py

```diff
@@ -681,6 +687,12 @@ def deselect_node(self, node):
             This method must be called by the derived widget using super if it
             is overwritten.
         '''
+        self._last_selected_node = None
+        self._reslove_last_node()
```

Comment:
Errors. Should be "self.resolve_last_node()"

Reply:
done